up-linux-streamer-plugin/DEFAULT_CONFIG.json5

```diff
@@ -13,17 +13,25 @@
         // Lower numbers mean that some messages will be dropped
         message_queue_size: 10000
       },
+      streamer_uuri: {
+        // Determines the authority_name of the host device
+        // Used when initializing host transport
+        authority: "linux",
+        // Determines the ue_id of the streamer
+        // Used when initializing host transport
+        ue_id: 78,
```

Comment:
since you are using json5, you might also want to write this as a hex number (as we do all across the Rust code) ...

Reply:
Good point